charts/cf-runtime/README.md

```diff
@@ -1,6 +1,6 @@
 ## Codefresh Runner
 
-![Version: 7.9.4](https://img.shields.io/badge/Version-7.9.4-informational?style=flat-square)
+![Version: 8.0.0](https://img.shields.io/badge/Version-8.0.0-informational?style=flat-square)
```

Comment:
@vitalii-codefresh Please add an upgrade guide with breaking changes description and list of actions that needed to be taken for smooth upgrade.